### PR TITLE
8

### PR DESCRIPTION
**Pull request by umaaug**

### CODE REVIEW

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md [hunk 1]
```diff
@@ -183,81 +183,74 @@ Manu II. 176 and Yāj. I. 99, 100, 102 show that _Devapūjā_ is different from[
...
+[^77]: For 38 उपचारs vide व्रतराज p. 44. 36 उपचारs are enumerated in एकादशीतत्त्व p. 58 quoting from आह्निकचिन्तामणि. They are not set out here for reasons of space. The 16 उपचारs quoted from प्रपञ्चसार by व.क्रि. कौ. P 156 and ए. त. p. 59 are : आसनं स्वागतं पाद्यमध्य॑माचमनीयकम् । मधुपर्काचमस्नान वसनाभरणानि च। सुगन्धसुमनोधूपदीपनैवेद्यवन्दनम् । प्रयोजयेदर्चनायामुपचारांश्च षोडश । अर्घ्यपाद्याचमनकमधुपर्काचमा अपि । गन्धादयो निवेद्यान्ता <U>उपचारा दश क्रमात्</U> । गन्धादयो निवेद्यान्ताः <U>पूजा पञ्चोपचारिकी</U>। उपचारैर्यथाशक्ति देवतामन्वहं यजेत् ।. The व.क्रि. को. (p. 157) proceeds : 'अत्यन्ताशक्तावुपचारद्वयेनापि पूजामाह कालिकापुराणे । ... अभावे <u>गन्धपुष्पाभ्यां तदभावे च भक्तितः</u>।'. The व. क्रि. को. p 156 notes that there is slight difference in the order in which some apacāras are offered 'अत्र अर्ध्यानन्तरं पाद्यमाचमनीयं चोक्तम् । शारदातिलकादिषु पाद्याचमनीयानन्तरमध्यर्मुक्तम् । कालिकापुराणे पाद्यार्थ्याचमनीयक्रम उक्तः ': Vide ब्रह्मवैवर्त (ब्रह्मखण्ड) chap. 26.90-92 for उपचारs.
```
Comment (vvasuki) [inline kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:188]: पाद्यमर्घ्यमाचमनीयकम्

**Review by vvasuki — COMMENTED** [review]
नमः। 

please correct and let me know. (You can type response under the comment and send.)

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md [hunk 2]
```diff
@@ -244,154 +244,151 @@ It is no doubt true that in most vratas brāhmaṇas are to be fed, but one shou
 
 [^90]: पुष्पाणां प्रवरा जाती धूपानां कुन्दको वरः। गन्धानां कुङ्कमं श्रेष्ठं लेपानां चन्दनं वरम् ॥ दीपदाने घृतं श्रेष्ठं मोदकाश्च निवेदने। भविष्यपुराण (ब्राह्मपर्व 198. 19 q. by कल्पतरु on व्रत p. 183 ).
 
-[^91]: विफलार्जुनपुष्पाणि भल्लातकशिरीष्कम्। लाक्षा सर्जरसश्चैव विडङ्श्चैव गुग्गुलु:। एतैर्धूपो मक्षिकाणां मशकानांं विनाशनः॥ गरुडपुराण I. 177. 88–89.
+[^91]: विफलार्जुनपुष्पाणि भल्लातकशिरीष्कम्। लाक्षा सर्जरसश्चैव पिडाङ्श्चैव गुग्गुलु:। एतैर्धूपो मक्षिकाणां मशकानांं विनाशनः॥ गरुडपुराण I. 177. 88–89.
```
Comment (vvasuki) [inline kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:247]: First correction is doubtful - can you send a screenshot on Telegram plz?

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md [hunk 3]
```diff
@@ -244,154 +244,151 @@ It is no doubt true that in most vratas brāhmaṇas are to be fed, but one shou
...
 
 There were certain differences between Vedic sacrifices and vratas. The promised reward of most sacrifices was heaven, while Purāṇas promised that most vratas would confer tangible benefits in this very life. Further, vratas could be performed by: all, including sūdras, maidens, married women, widows, even prostitutes.
 
-There were vratas for the brahmaçõrin (called Veda-vratas ) and for the spātaka. These have been already dealt with in H. of Dh. vol. II, pp. 370-374 and pp. 412-415 respectively. That
+There were vratas for the _brahmaçõrin_ (called Veda-vratas ) and for the sṇātaka. These have been already dealt with in H. of Dh. vol. II, pp. 370-374 and pp. 412-415 respectively.
```
Comment (vvasuki) [inline kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:335]: brahmacārin , snātaka

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md [hunk 4]
```diff
@@ -244,154 +244,151 @@ It is no doubt true that in most vratas brāhmaṇas are to be fed, but one shou
...
+That vratas (in the medieval sense ) existed even in the first centuries before and after the Christian era follows from the Āp. Dh. S. and from the dramas of Kālidāsa, from the Mṛcchakaṛika and the Ratnāyali. The Āp. Dh. S. II. 8. 18 prescribes a vrata for one year which is equal in its merit to brahmacārivrata for 48 years and Ap. Dh. S. II. 8. 18-19 to Āp. Dh. S. II. 8. 20 prescribe two modes of a vrata to be begun on Tisya (Puśya) naksatra to be observed by a person desirous of prosperity. Vide S. B. E. vol. II, pp. 145– 148. Hemādri (vrata II. .p. 628 quotes Āp. Dh. S. II. 8. 20.3-9), explains the sutras and calls this Āpastambokta Pusyavrata.
```
Comment (vvasuki) [inline kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:345]: Mṛcchakaṭika

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md [hunk 5]
```diff
@@ -244,154 +244,151 @@ It is no doubt true that in most vratas brāhmaṇas are to be fed, but one shou
...
 
-115. गृहीतव्रताकरणे छागलेयः। यो गृहीत्वा व्रतं मोहासाचरेत्काममोहितः। जीवग्भवति
 
-E a: Tor Jr Hq. . P. 784, 7. fil. fat. p. 9, a p. 153 (quoting from प्राय. विवेक), नि. सि. P. 26 (quoting from मदनरत्न and reading
+[^114]: It has to be remembered that several different vratas are men. tioned under the same name, also that different dames are given to the same vrata and that some śantis, utsavas and sṇanas also are described in works on vratas.
```
Comment (vvasuki) [inline kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:362]: snānas

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 1 above)
Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:188]: corrected.

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 2 above)
Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:247]: at present I have put पिडाङ्श्चैव{??} like this. I will correct it once we get the correct word.

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 3 above)
Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:335]: corrected.

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 2 above)
Comment (vvasuki) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:247]: I looked up garuDapurANa. Please put विडङ्गश्चैव

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 1 above)
Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:188]: [^77]: For 38 उपचारs vide व्रतराज p. 44. 36 उपचारs are enumerated in एकादशीतत्त्व p. 58 quoting from आह्निकचिन्तामणि. They are not set out here for reasons of space. The 16 उपचारs quoted from प्रपञ्चसार by व.क्रि. कौ. P 156 and ए. त. p. 59 are : आसनं स्वागतं पाद्यमर्घ्यमाचमनीयकम् । मधुपर्काचमस्नान वसनाभरणानि च। सुगन्धसुमनोधूपदीपनैवेद्यवन्दनम् । प्रयोजयेदर्चनायामुपचारांश्च षोडश । अर्घ्यपाद्याचमनकमधुपर्काचमा अपि । गन्धादयो निवेद्यान्ता <U>उपचारा दश क्रमात्</U> । गन्धादयो निवेद्यान्ताः <U>पूजा पञ्चोपचारिकी</U>। उपचारैर्यथाशक्ति देवतामन्वहं यजेत् ।. The व.क्रि. को. (p. 157) proceeds : 'अत्यन्ताशक्तावुपचारद्वयेनापि पूजामाह कालिकापुराणे । ... अभावे <u>गन्धपुष्पाभ्यां तदभावे च भक्तितः</u>।'. The व. क्रि. को. p 156 notes that there is slight difference in the order in which some apacāras are offered 'अत्र अर्ध्यानन्तरं पाद्यमाचमनीयं चोक्तम् । शारदातिलकादिषु पाद्याचमनीयानन्तरमध्यर्मुक्तम् । कालिकापुराणे पाद्यार्थ्याचमनीयक्रम उक्तः ': Vide ब्रह्मवैवर्त (ब्रह्मखण्ड) chap. 26.90-92 for उपचारs.

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 3 above)
Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:335]: There were vratas for the _brahmcārin_ (called Veda-vratas ) and for the snātaka. These have been already dealt with in H. of Dh. vol. II, pp. 370-374 and pp. 412-415 respectively.

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 4 above)
Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:345]: That vratas (in the medieval sense ) existed even in the first centuries before and after the Christian era follows from the Āp. Dh. S. and from the dramas of Kālidāsa, from the Mṛcchakaṭika and the Ratnāyali. The Āp. Dh. S. II. 8. 18 prescribes a vrata for one year which is equal in its merit to brahmacārivrata for 48 years and Ap. Dh. S. II. 8. 18-19 to Āp. Dh. S. II. 8. 20 prescribe two modes of a vrata to be begun on Tisya (Puśya) naksatra to be observed by a person desirous of prosperity. Vide S. B. E. vol. II, pp. 145– 148. Hemādri (vrata II. .p. 628 quotes Āp. Dh. S. II. 8. 20.3-9), explains the sutras and calls this Āpastambokta Pusyavrata.
Corrected.

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 5 above)
Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:362]: [^114]: It has to be remembered that several different vratas are mentioned under the same name, also that different names are given to the same vrata and that some śantis, utsavas and snānas also are described in works on vratas.
Corrections made.

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 4 above)
Comment (vvasuki) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:345]: Please correct in the file and push.

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 5 above)
Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:362]: [^114]: It has to be remembered that several different vratas are mentioned under the same name, also that different names are given to the same vrata and that some śantis, utsavas and snānas also are described in works on vratas.
Corrected.

Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:362]: corrected

File: kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md (hunk 2 above)
Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:247]: corrected.
[^91]: विफलार्जुनपुष्पाणि भल्लातकशिरीष्कम्। लाक्षा सर्जरसश्चैव विडङ्गश्चैव गुग्गुलु:। एतैर्धूपो मक्षिकाणां मशकानांं विनाशनः॥ गरुडपुराण I. 177. 88–89.

Comment (umaaug) [reply kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md:247]: corrected. [^91]: विफलार्जुनपुष्पाणि भल्लातकशिरीष्कम्। लाक्षा सर्जरसश्चैव विडङ्गश्चैव गुग्गुलु:। एतैर्धूपो मक्षिकाणां मशकानांं विनाशनः॥ गरुडपुराण I. 177. 88–89.

**Pull request merged**